third-party/linshare/src/test/java/org/apache/james/linshare/Linshare.java

```diff
@@ -133,4 +139,14 @@ private GenericContainer createDockerBackend() {
                 .withStartupTimeout(Duration.ofMinutes(10)))
             .withNetwork(network);
     }
+
+    public RequestSpecification fakeSmtpRequestSpecification() {
```

Comment:
(Just an opinion) you can use feign client to get a POJO responses from Fake SMTP, it's helps assertions look better.

Reply:
Hm I would agree if we had more than one request towards Fake SMTP but it's not the case, so I would still need to declare a Feign.builder() with similar params here. Maybe not necessary?

Reply:
it's up to you :) It might require a big refactoring